packages/core/src/components/BaseInput/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as BaseInput } from "./BaseInput";
```

Comment:
This shouldn't be exported for consumers of the lib, does it mean now it does export it?

Reply:
It's only exporting the content of the folder, I haven't included it in the components' index file, so it's relevant only internally